src/bin/flamegraph.rs

```diff
@@ -174,6 +174,14 @@ struct Opt {
     )]
     search_color: SearchColor,
 
+    /// Stroke color
```

Comment:
We should document how this affects the design ("adds an outline to every frame") as well as the format here too I think. Probably also the stroke width.

Reply:
I'm not sure it's necessary to define the format here, it isn't done for the other options. As for the stroke width I also don't think we should mention it either, it doesn't add anything in my opinion.

Changed the text to `Adds an outline to every frame`.